### DISCUSSION TITLE
Add example dag showing usage of `impersonation_chain` argument of Google operators

### DISCUSSION THREAD

**Discussion by olchas** [apache/airflow#26650] [General]
**Description**

This issue is about adding an example dag showing usage of `impersonation_chain` argument of Google operators. It should show both using this argument as a string and as a sequence.

**Use case / motivation**

This is to provide users of Google operators with instructions on how to use [direct impersonation](https://cloud.google.com/iam/docs/understanding-service-accounts#directly_impersonating_a_service_account) with them.

**Related Issues**
https://github.com/apache/airflow/issues/8803

**Comment by sharad-vm** [comment]
I can take a shot at this. @eladkal, would you be able to assign this to me, please?

**Comment by mik-laj** [comment]
@sharad-vm I assigned you to this ticket.

**Comment by mik-laj** [comment]
@sharad-vm  Do you need help? I will be happy to help you.

**Comment by sharad-vm** [comment]
Thanks for the follow-up and offering to help, @mik-laj!
I will let you know in 24 hours.

**Comment by sharad-vm** [comment]
Actually @mik-laj, I might need some help. 
I understand that the 'impersonation_chain' argument has been added to the files in airflow/providers/google/cloud/hooks/...

I see example_dags here - airflow/airflow/providers/google/cloud/example_dags/...
Is this where I add an example for all the 49 files changed?

**Comment by mik-laj** [comment]
You should create a new example dag. you just need to run it once and see if it works.  It can do anything, such as create and delete a dataset. In the next step I will add system tests to this example DAG, but it is extra effort.

**Comment by sharad-vm** [comment]
Thanks! Will open a PR this weekend.

**Comment by eladkal** [comment]
@sharad-vm are you working on this issue?

**Comment by rightx2** [comment]
Reeaaaaaally need it

**Comment by thomashutcheson-msm** [comment]
Did the example dag get created, I would find this really useful right now ?

**Comment by sharad-vm** [comment]
@thomashutcheson-msm , @rightx2  - Apologies. I lost track of this one. I will circle back to this by Wednesday this week and give you all an update here. Thank you!
